test/unit/BraveAlerterConfiguratorTest/getHistoricAlertsByAlertApiKeyTest.js

```diff
@@ -58,7 +58,7 @@ describe('BraveAlerterConfigurator.js unit tests: getHistoricAlertsByAlertApiKey
       id: 'id2',
       display_name: 'displayName2',
       incident_type: 'incidentType2',
-      alert_type: ALERT_TYPE.SENSOR_NO_MOTION,
+      alert_type: ALERT_TYPE.SENSOR_STILLNESS,
```

Comment:
Oops, I somehow missed this one during the last Pull Request. Funny that it still passed with it being undefined.